examples/vllm-grafana-dashboard.json

```diff
@@ -0,0 +1,1102 @@
+{
```

Comment:
Lets put this in `examples/observability/` to follow the current subdir convention in `examples/`.

Reply:
done